### ISSUE TITLE
FightPandemic - Full Stack Engineer - Remote Worldwide

### ISSUE THREAD

**Issue by letizia123** [TechnologyMasters/jobs#325]
### Salary Expectation

- Volunteer

### Benefits

Why Join Us?

- Expand your network. You get to work as a part of a innovative, global and diverse team.
- Be part of a movement. We strive for bold changes and envision to make a big impact. Bring your passion, energy and creativity to participate.
- Learn and grow. You get to continuously learn and develop your skills. You might join us because of your skills in business development, but are able to contribute to any team's work if you'd like.
- Be free. You have the freedom to set the hours you are able to work every week.
- Do good. It is a scientific fact that volunteering for a good cause improves your well being.

### Location

- Volunteer

## What You'll Do _(Job Description)_

- Develop features and improvements to the Fight Pandemics product in a secure, well-tested, and performant way.
- Collaborate with other engineers, product managers and other stakeholders (e.g. UX) to maintain a high bar for quality in a fast-paced, iterative environment.
- Take ownership of the complete lifecycle of a product features from planning to implementation to testing to production.
- Build and operate high-quality infrastructure and services.
- Craft code that meets our internal standards for style, maintainability, and best practices for a high-scale web environment. Maintain and advocate for these standards through code review.
- Solve technical problems of moderate scope and complexity and help scale our infrastructure.

### Must Have

Qualifying Skills:

- You are passionate about what we are building and you want to contribute and not wait on the sideways.
- You are positive and have a solution-oriented mindset.
- You are a problem solver and enjoy writing robust code
- You have attention to detail with strong communication skills.
- Knowledgable about building full-stack web applications using HTML5, CSS3, and JavaScript related technologies such as React and Node.js.
- Knowledgable about building APIs that are RESTful.
- Knowledgeable about SQL and NoSQL databases and building efficient data queries and pipelines.
- Knowledgeable about building secure APIs that are not vulnerable to security attacks.
- You know how to use basic Git commands.
- Most importantly, you want to be part of something great that positively impacts society especially in times of crisis.

### Nice to Have

- Experience working with our stack
Frontend: React Hooks, Ant Design, Ant Design Mobile, styled-components.
Backend: Fastify, MongoDB, Auth0, Google Maps API.
DevOps: Docker
- Developer platform/tool industry experience

### Relevant Experiences

- Openness to learn new technologies.
- Experience working with Git and Github in teams.
- Experience working with a remote team.
- Open source experience.

## About Our company

We are creating an altruistic platform that facilitates the connection and exchange of information between individuals and/or organizations that need help with those that are providing help and vice versa. The way that we are going to bootstrap our social media, is by listing thousands of projects and initiatives that are tackling COVID-19 pandemic.
We are looking for volunteer who want to join us in our fight against pandemics like COVID-19 and other crisis alike.

## How to apply

Please apply here -> https://apply.workable.com/fightpandemics/j/B17D029D45/

---

## Meta

- [ ] Full Time
- [ ] Part Time
- [ ] Contract
- [ ] Internship
- [ ] Remote Worldwide
- [ ] Remote Regional
- [ ] Remote OK

**Comment by cr03** [comment]
Sorry, but we don't allow the promotion of non-paying opportunities, because "a job", and "employment" have strict legal definitions: https://www.ontario.ca/document/your-guide-employment-standards-act-0/employee-status#section-4

I understand you may only be looking for volunteers, in which case, would you kindly feel free to close your volunteer issues and use the #showcase channel on our Slack Chat?

Best regards.

**Comment by cr03** [comment]
https://github.com/TechnologyMasters/jobs/issues/325
https://github.com/TechnologyMasters/jobs/issues/324
https://github.com/TechnologyMasters/jobs/issues/323
https://github.com/TechnologyMasters/jobs/issues/322

**Comment by letizia123** [comment]
Dear Ian,

Many thanks for your email.

I am sorry for publishing the volunteer opportunities on the #job channel,
I will delete the tickets and publish them on the showcase one, thank you
for your understanding.

Best regards,
Letizia

Il giorno ven 28 ago 2020 alle ore 22:55 Ian Penney <
notifications@github.com> ha scritto:

**Comment by theetrain** [comment]
Closing due to missing information.
